.devcontainer/postCreate.sh

```diff
@@ -3,4 +3,5 @@
 # Turn off git status check to improve zsh response speed: https://stackoverflow.com/a/25864063
 git config --add oh-my-zsh.hide-status 1
 git config --add oh-my-zsh.hide-dirty 1
+pre-commit install -t pre-commit -t prepare-commit-msg
```

Comment:
Manually verified that the pre-commit hooks are enabled:

![Uploading Screen Shot 2022-11-16 at 13.38.49.png…]()

Reply:
This script is executed after container creation as the name indicates.